eclair-core/src/main/resources/reference.conf

```diff
@@ -31,6 +31,7 @@ eclair {
 
   min-feerate = 3 // minimum feerate in satoshis per byte
   smooth-feerate-window = 6 // 1 = no smoothing
+  feerate-provider-timeout = 1 second // max time we'll wait for answers from a fee provider before we fallback to the next one
```

Comment:
Timeout is really `1 second`, not `5 seconds`.

Reply:
I think we should set 5 seconds here and no default value in the object constructors

Reply:
Agreed.